TheMovieDb/ViewController.swift

```diff
@@ -7,13 +7,40 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UICollectionViewDelegate {
+    let collectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
```

Comment:
instead use a CollectionViewController it will reduce the amount of work you need for this